volatility/framework/symbols/mac/extensions/__init__.py

```diff
@@ -465,3 +465,30 @@ def get_address(self):
 
         return ip
 
+class sysctl_oid(objects.StructType):
+    def get_perms(self):
```

Comment:
Sorry, I know this has gone through multiple revisions, but could you please include docstrings and type information (it looks like `-> str` in both cases)?

Reply:
I will get the doc strings added and do a new commit